res/img/icon-pill-remove.svg

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Same export issues here as well.

Reply:
This was the SVG supplied by Nad, though instead of going through inkscape it was shoved through a different (less useful) tool.